spec/mailers/feedback_mailer_spec.rb

```diff
@@ -58,6 +58,10 @@
       it 'has the last search' do
         expect(mail.body).to have_content '/?q=kittenz'
       end
+
+      it 'has the current time' do
+        expect(mail.body).to have_content "Time sent: #{Time.zone.now}"
```

Comment:
I originally ran this a few times and couldn't get it to error out (but did so because I was concerned about that exact issue).

Do you have a suggestion of a better way to go about writing this test (or just drop it)?

Reply:
Updated to mock the current time